lib/components/home/artwork_rail.js

```diff
@@ -50,6 +58,7 @@ export default Relay.createContainer(ArtworkRail, {
         results @include(if: $fetchContent) {
           __id
           title
```

Comment:
Oh I meant just for the artworks, I didn’t even realise that the `__id` for the rails could also be removed :)